polytope/polytope.py

```diff
@@ -201,19 +201,40 @@ def __copy__(self):
         P.fulldim = self.fulldim
         return P
 
+    def contains(self, point, abs_tol=ABS_TOL):
+        """Return True if polytope contains points.
+
+        @type point: array_like, 1d array, or 2d array (of column vectors)
+
+        @rtype: bool, 1d array
+
+        See Also
+        ========
+        L{is_inside}
+        """
+        if not isinstance(point, np.ndarray):
+            point = np.array(point)
+        if point.ndim == 1:
+            point.shape = (point.size, 1)
+        assert point.shape[0] == self.dim, "points should be col vectors"
```

Comment:
Writing "column" may be easier to understand for unfamiliar users. However, writing too much elongates the line. One way to keep the line short, but have a message of arbitrary length is:

```python
assert foo, (
    'points should be column vectors, '
    '...')
```

Also, it is worth mentioning that in optimized mode Python does not check `assert` statements. One may raise an exception instead. I strongly prefer the assertion-based style, because it is short, readable, and to the point. In contrast, each exception check would require at least two lines, and clutter the code with `if` statements, distracting from the logical structure of the task.